mmdata.c

```diff
@@ -1548,6 +1609,21 @@ nmbrString *nmbrAddElement(nmbrString *g, long element)
 }
 
 
+/* Add a single number to start of a nmbrString - faster than nmbrCat */

```

Comment:
The return value is pushed on an internal variable stack with an implicit memory management.  This important detail is not mentioned in the comment.